docs/modules/ROOT/pages/how-tos/aks_recover_from_backup.adoc

```diff
@@ -0,0 +1,141 @@
+= AKS Recover cluster objects from Azure Blob Storage backup
+
+== Prerequisites
+
+* Executables used in this guide:
+** `kubectl`
+** `yq`
+** `restic`
+* API access to the target cluster
+
+== General procedure
+
+1. Collect configuration for restic
+2. Identify and retrieve restic snapshot
+3. Extract files containing the desired objects and prepare them
+4. Apply objects to the cluster
+
+== Collect restic configuration
+
+Restic requires the environment variables `RESTIC_REPOSITORY`, `RESTIC_PASSWORD`, `AZURE_ACCOUNT_NAME` and `AZURE_ACCOUNT_KEY` to be set.
+They can be obtained from the target cluster itself.
+They can also be obtained from the cluster catalog and Vault.
+
+=== Obtaining restic configuration from cluster
+
+[source,console]
+----
+export RESTIC_REPOSITORY=$(kubectl -n syn-cluster-backup get schedule objects -o jsonpath='azure:{.spec.backend.s3.bucket}:/')
+export RESTIC_PASSWORD=$(kubectl -n syn-cluster-backup get secret objects-backup-password -o jsonpath='{.data.password}' | base64 --decode)
+export AZURE_ACCOUNT_NAME=$(kubectl get secret admin-credentials-minio-k8up -n syn-minio -o jsonpath='{.data.accesskey}' | base64 --decode)
+export AZURE_ACCOUNT_KEY=$(kubectl get secret admin-credentials-minio-k8up -n syn-minio -o jsonpath='{.data.secretkey}' | base64 --decode)
+----
+
+=== Obtaining restic configuration from catalog and vault
+
+1. Obtain the repository URL of the clusters catalog and export it to `REPO_URL`
++
+Get the URL from https://control.vshn.net/syn/lieutenantclusters.
+Alternatively, get it from the Lieutenant API or the Kubernetes API Lieutenant is running on.
++
+.Use the Lieutenant API
+[source,console]
+----
+REPO_URL=$(curl -sH "${LIEUTENANT_AUTH}" "https://${LIEUTENANT_URL}/clusters/${CLUSTER_ID}" | jq -r .gitRepo.url)
+----
++
+.Use the Kubernetes API
+[source,console]
+----
+REPO_URL=$(kubectl -n ${LIEUTENANT_NS} get cluster -o jsonpath='{.spec.gitRepoURL}' ${CLUSTER_ID})
+----
+
+2. Download and extract the cluster catalog
++
+[source,console]
+----
+mkdir catalog
+git archive --remote ${REPO_URL} master | tar -xC catalog
```

Comment:
What's the reasoning for preferring `git archive` over a regular `git clone` here? I don't really mind either way, just curious.

Reply:
Copy pasta from the OCP4 variant of this documentation appuio/openshift4-docs#212ef5f44c04be8e65e4d3c15bec4b907c8ada0 Didn't reflect on that.